src/Traits/Configurable.php

```diff
@@ -83,6 +83,12 @@ private function configApi()
 
 	private function getFileName()
 	{
+		$user = auth()->user();
+
+        	if ($user) {
```

Comment:
Same here, if the config variable is set to true and also you're missing the `return` statement